### DISCUSSION TITLE
How to make it run fullscreen?

### DISCUSSION THREAD

**Discussion by lida222** [sultim-t/xash-rt#138] [Q&A]
Setting my native monitor resolution does nothing.  I have also tried alt + enter.  Any ideas?

**Comment by stephen-cusi** [comment]
Adjust in options

**Reply by lida222** [reply]
I have set my native resolution to what is listed in game.  Still no fullscreen.  It runs in a window no matter what i do.  If i maximize the window, it still shows the window and my desktop background.

**Comment by Nord1cWarr1or** [comment]
set `-fullscreen` in launch parameters
